bdd_tests/features/smoketests.feature

```diff
@@ -7,21 +7,46 @@ Feature: Basic set of smoke tests
      Then I should find that file on PATH
 
 
-  Scenario: Check if CCX Notification Writer displays help message
+  Scenario: Check if Kafkacat utility is available
     Given the system is in default state
-     When I start the CCX Notification Writer with the --help command line flag
-     Then I should see help messages displayed on standard output
+     When I look for executable file kafkacat
+     Then I should find that file on PATH
 
 
-  Scenario: Check if CCX Notification Writer displays authors
+  Scenario: Check if jps utility is available
     Given the system is in default state
-     When I start the CCX Notification Writer with the --authors command line flag
-     Then I should see info about authors displayed on standard output
+     When I look for executable file jps
+     Then I should find that file on PATH
 
 
   Scenario: Check if Postgres database is available
-    Given the system is in default state
-     When I connect to database named test as user postgres with password postgres
+    Given Postgres is running
+     When I connect to database named postgres as user postgres with password postgres
      Then I should be able to connect to such database
      When I close database connection
      Then I should be disconnected
+
+
+  Scenario: Check if CCX Notification Writer database can be reached
```

Comment:
I don't see the value of this scenario

Reply:
it's just a smoke pre-test for other scenarios, but you are right - it actually do the check in "Given" clause :)